case foobar() of
	{ok, NumberReturned} -> doSomethingWith(NumberReturned);
        {error, ErrReason} -> logError(ErrReason)
end.